java/src/jmri/implementation/AbstractMemory.java

```diff
@@ -39,6 +42,17 @@ public void setValue(Object v) {
         // notify
         firePropertyChange("value", old, _current);
     }
+    
+    /**
+     * {@inheritDoc} 
+     * 
+     * Do a string comparison.
+     */
+    @CheckReturnValue
+    @Override
+    public int compareSystemNameSuffix(@Nonnull String suffix1, @Nonnull String suffix2, @Nonnull NamedBean n) {
+        return suffix1.compareTo(suffix2);
```

Comment:
Why does the default implementation in NamedBean not work?

Reply:
See the comment above.